src/services/chatgpt.ts

```diff
@@ -0,0 +1,88 @@
+const BASE_URL = 'https://api.openai.com/v1'
```

Comment:
Move the API key and base URL to a separate configuration file or environment variables to enhance security and maintainability.